styles.css

```diff
@@ -288,3 +285,84 @@ button:hover,
 }
 
 /* About myself section */
+
+/* contact section */
+.contact-section {
+  background-color: #fff;
+  margin-left: -2.4rem;
+  margin-right: -3.2rem;
+}
+
+.contact-wrapper {
+  background-image: url(images/contact-form-shapes-mobile.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+  background-color: #6070ff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-top-left-radius: 5.6rem;
+  padding: 5rem 1.5rem;
+}
+
+.contact-section h1 {
+  margin-top: 20%;
+  color: #fff;
+  font-size: 2.5rem;
+  line-height: 3.25rem;
+  font-weight: 700;
+  margin-bottom: 1rem;
+  text-align: center;
+}
+
+.contact-section p {
+  color: #ebebff;
+  font-size: 1.25rem;
+  font-weight: 400;
+  line-height: 1.75rem;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem 0;
+  width: 100%;
+}
+
+form input[type="text"],
+input[type="email"],
+input[type="submit"],
+form textarea {
+  border-radius: 0.5rem;
+  padding: 1rem;
+  margin: 0.7rem 0;
+  border: 0;
+}
```

Comment:
- [ ] Nice work so far 👍🏿 I notice on the hover of the button background color seems to be different from the Figma design, I kindly suggest you replace the present color with `#6070ff` as required by the design

Reply:
## I did it as figma design
- --tag-button-color: #6070ff; online 
![image](https://github.com/mulugithub/Microverse-Portfolio/assets/61753075/0e26d1fe-514e-4cdc-b1df-958396a69879)
-The output the same as figma design when mouse hover for tags buttons except submit button in the contact section
![image](https://github.com/mulugithub/Microverse-Portfolio/assets/61753075/6a54b1f0-983e-4a75-861e-fbc955ac323f)
- In the contact section I changed the background. because the background color for contact form is  #6070ff

Reply:
Please check the get in touch button on hover. That's where the issue is.

Reply:
Thank you sir